ls1intum/Artemis
# [Exam] Hand in early client fix

### Checklist

### Motivation and Context
See https://github.com/ls1intum/Artemis/issues/1903

### Description
- [x] Add a guard before saving the submissions 
- [x] Set the claimed working time by the student on submit

### Steps for Testing
1. Start an already created & started exam on two different browsers as the same student.
2. Answer the questions on one browser and hand in the exam early. Get directed to the exam summary page.
3. Answer questions on the other browser differently, try to hand in it early, it shouldn't work.
4. After the deadline passes refresh the page, you should again be redirected to the exam summary page.
5. You should be able to see two different summaries & exports available for the same student on two different browsers.

### Screenshots
